tests/project_root_flag/foo/src/foo.toit

```diff
@@ -0,0 +1,3 @@
+// Copyright (C) 2020 Toitware ApS. All rights reserved.
```

Comment:
We should try to make sure all the tests have the same copyright header that includes the licensing information.

Reply:
Completely agree.
I will change the copyrights once the migration is complete.